dll/kernel32.cpp

```diff
@@ -633,6 +633,22 @@ namespace kernel32 {
 		}
 	}
 
+	DWORD WIN_FUNC GetTempPathA(DWORD nBufferLength, LPSTR lpBuffer) {
+		DEBUG_LOG("GetTempPathA\n");
+
+		if ((nBufferLength == 0) || (lpBuffer == 0)) {
+			return 0;
+		}
+
+		const char* tmp_dir;
+		if (!(tmp_dir = getenv("WIBO_TMP_DIR"))) {
+			tmp_dir = "Z:\\tmp";
+		}
```

Comment:
According to the docs the path should end with a backslash
https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettemppatha
```suggestion
		if (!(tmp_dir = getenv("WIBO_TMP_DIR"))) {
			tmp_dir = "Z:\\tmp\\";
		}
```